enumerables.rb

```diff
@@ -0,0 +1,199 @@
+# rubocop:disable Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity, Metrics/MethodLength, Style/RedundantSelf, Style/GuardClause, Style/IfUnlessModifier, Metrics/BlockNesting, Metrics/ModuleLength
```

Comment:
- [ ] So many cops are disabled, please fix the errors instead of disabling the cops.

- [NOTE] Style/CaseEquality cop can be disabled.